lib/internal/errors.js

```diff
@@ -129,6 +129,8 @@ E('ERR_INVALID_OPT_VALUE',
   (name, value) => {
     return `The value "${String(value)}" is invalid for option "${name}"`;
   });
+E('ERR_INVALID_REPL_EVAL_CONFIG',
+  'Cannot specify both breakEvalOnSigint and eval for REPL');
```

Comment:
It's ok :) It was labelled as such even before the 8.0 came out so I guess it kind of stuck there...